projects/fribidi/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -eu
+# Copyright 2016 Google Inc.
```

Comment:
Please update the license to `Copyright 2021 Google LLC`

Reply:
Done